src/components/UI/Card.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from "react"
```

Comment:
```suggestion
import type { ReactNode } from "react"
```

타입만 쓰기위해 import 해온 값이라면 `import type`을 쓰는 것을 권장 드립니다.

- 코드 분석 시 type인지 값인지 명시적으로 구분하기 위함입니다.

Reply:
넵 알겠습니다!!